boomer/run.go

```diff
@@ -63,8 +66,10 @@ func (b *Boomer) worker(wg *sync.WaitGroup, ch chan *http.Request) {
 		size := int64(0)
 		resp, err := client.Do(req)
 		if err == nil {
-			size = resp.ContentLength
-			code = resp.StatusCode
+			if _, err = io.Copy(ioutil.Discard, resp.Body); err == nil {
+				size = resp.ContentLength
+				code = resp.StatusCode
```

Comment:
These response values are still valid even if reading the body fails. Why would we want exclude them from the results?